ERCS/erc-4377.md

```diff
@@ -0,0 +1,361 @@
+---
+eip:
+title: A Protocol for ERC20 Tokens Which Released Based On Predefined Periods Controlled by Smart Contracts
```

Comment:
Suggest shortening the title, perhaps something like "Release ERC20 tokens on predefined periods"